packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -6,7 +6,7 @@ import { TableCell } from '../cell';
 
 // prettier-ignore
 export const template = html<TableRow>`
-    <template role="row">
+    <template role="row" style="grid-template-columns: ${x => x.rowGridColumns};">
```

Comment:
One thought I had (but didn't try out): could we move this `grid-template-columns` style up to the `.table-row-container` on the table template, and then just have a style like `grid-template-columns: inherit` in the TableRow styles? (Might need to do it with CSS variables).

With the current approach every row gets an inline style with the exact same `grid-template-rows` value which feels somewhat verbose. Not really a big issue but you see it in DevTools:
![TableRowGridTemplateColumns](https://user-images.githubusercontent.com/20709258/222831430-162cdf88-a4d8-46c5-8294-1df65fe8655e.PNG)

Reply:
Done.